src/Catty/ViewController/Help/HelpWebViewController.swift

```diff
@@ -229,10 +229,27 @@ class HelpWebViewController: UIViewController, WKUIDelegate, WKNavigationDelegat
             return
         }
 
-        let url = URL(string: urlWithoutParams ?? "")
+        let storeProjectDownloader = StoreProjectDownloader.init()
         if let fileManager = CBFileManager.shared() {
             fileManager.delegate = self
-            fileManager.downloadProject(from: url, withProjectID: projectID, andName: projectName)
+
+            storeProjectDownloader.download(
```

Comment:
I think there is a problem with this code. Apparently the URL has changed on the web, so the download does not work anymore, but also if it does, I think the actual project does not get stored on device.